src/storage/keyvalue/JsonResourceStorage.ts

```diff
@@ -127,4 +140,8 @@ export class JsonResourceStorage<T> implements KeyValueStorage<string, T> {
     // on the `entries` results matching a key that was sent before.
     return ensureLeadingSlash(identifier.path.slice(this.container.length));
   }
+
+  private applyHash(key: string): string {
+    return createHash('md5').update(key).digest('hex');
```

Comment:
Even though the odds are very low, should we perhaps use a better hashing algorithm to prevent collisions? The size is fixed anyway.